todo_app/app.py

```diff
@@ -21,39 +21,45 @@ def index():
 def render_cards(id):
     status = False
     cardsList = get_cards(id)
-    print(cardsList)
+    cardsList.sort(key=lambda x: x.status)
+    boardLists = get_boardLists(id)
 
-    return render_template("index.html", cardsList=cardsList, status=status)
+    return render_template(
+        "index.html",
+        cardsList=cardsList,
+        status=status,
+        boardLists=boardLists,
+        boardId=id,
+    )
 
 
-@app.route("/", methods=["POST"])
+@app.route("/add", methods=["POST"])
 def add_new_item():
-    item = request.form["item"]
-    add_item(item)
-    return redirect("/")
+    boardId = request.form.get("board_id")
+    inputItem = request.form.get("inputItem")
+    selectedList = request.form.getlist("selectedList")[0]
```

Comment:
This is a nice solution to a potentially unnecessary problem - is this a sign we should have used radio buttons instead of checkboxes for the lists so that we can confidently expect this to just contain a single value?